unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/PrivateChatWindow/PrivateChatWindowController.cs

```diff
@@ -311,4 +325,36 @@ private void HandleChatInputTriggered(DCLAction_Trigger action)
         if (!View.IsActive) return;
         chatHudController.FocusInputField();
     }
+
+    internal void RequestOldConversations()
+    {
+        if (isRequestingOldMessages)
+            return;
+
+        isRequestingOldMessages = true;
+        View?.SetOldMessagesLoadingActive(true);
+
+        if (!lastTimestampRequestedByUser.ContainsKey(ConversationUserId))
+            lastTimestampRequestedByUser.Add(ConversationUserId, DateTimeOffset.UtcNow.ToUnixTimeMilliseconds());
+
+        List<ChatMessage> currentPrivateMessages = chatController.GetAllocatedEntries()
```

Comment:
perhaps it would be better to ask the chatController all the allocated private messages for a specific user instead of doing the operation here